dll/cpl/console/console.c

```diff
@@ -237,10 +237,14 @@ InitApplet(HANDLE hSectionOrWnd)
     psh.dwFlags = PSH_PROPSHEETPAGE | PSH_PROPTITLE | /* PSH_USEHICON | */ PSH_USEICONID | PSH_NOAPPLYNOW | PSH_USECALLBACK;
 
     if (ConInfo->ConsoleTitle[0] != UNICODE_NULL)
+    {
         StringCchPrintfW(szTitle, ARRAYSIZE(szTitle), L"\"%s\"", ConInfo->ConsoleTitle);
+        psh.pszCaption = szTitle;
+    }
     else
-        StringCchCopyW(szTitle, ARRAYSIZE(szTitle), L"ReactOS Console");
```

Comment:
Note, to be explicit:
en-US has 'Console' only.

Reply:
Good point; I assume it's fine as such (i have no strong objection on this anyway)